schema.sql

```diff
@@ -0,0 +1,175 @@
+-- Schema yeticave_db
+-- -----------------------------------------------------
+
+-- -----------------------------------------------------
+-- Schema yeticave_db
+-- -----------------------------------------------------
+CREATE SCHEMA IF NOT EXISTS `yeticave_db` DEFAULT CHARACTER SET utf8 ;
+USE `yeticave_db` ;
+
+-- -----------------------------------------------------
+-- Table `yeticave_db`.`category`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `yeticave_db`.`category` (
+                                                        `category_id` INT NOT NULL AUTO_INCREMENT,
+                                                        `category_title` VARCHAR(100) NOT NULL,
+                                                        `symbolic_code` VARCHAR(100) NOT NULL,
+                                                        PRIMARY KEY (`category_id`),
+                                                        UNIQUE INDEX `title_UNIQUE` (`category_title` ASC) VISIBLE)
+    ENGINE = InnoDB;
+
+
+-- -----------------------------------------------------
+-- Table `yeticave_db`.`item`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `yeticave_db`.`item` (
+                                                    `item_id` INT NOT NULL AUTO_INCREMENT,
+                                                    `creation_date` TIMESTAMP(14) NOT NULL,
+                                                    `item_title` VARCHAR(100) NOT NULL,
+                                                    `description` VARCHAR(255) NOT NULL,
+                                                    `image` VARCHAR(255) NOT NULL,
+                                                    `start_price` INT NOT NULL,
+                                                    `completion_date` TIMESTAMP(14) NOT NULL,
+                                                    `bet_step` INT NOT NULL,
+                                                    PRIMARY KEY (`item_id`))
+    ENGINE = InnoDB;
+
+
+-- -----------------------------------------------------
+-- Table `yeticave_db`.`bet`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `yeticave_db`.`bet` (
+                                                   `bet_id` INT NOT NULL AUTO_INCREMENT,
+                                                   `bet_date` TIMESTAMP(14) NOT NULL,
+                                                   `total` DECIMAL(10,2) NOT NULL,
+                                                   PRIMARY KEY (`bet_id`))
+    ENGINE = InnoDB;
+
+
+-- -----------------------------------------------------
+-- Table `yeticave_db`.`user`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `yeticave_db`.`user` (
+                                                    `iser_id` INT NOT NULL AUTO_INCREMENT,
+                                                    `registration_date` TIMESTAMP(14) NOT NULL,
```

Comment:
[ERROR in query 9] Too-big precision 14 specified for 'registration_date'. Maximum is 6.